### PR TITLE
Add SPU XFloat Accuracy combobox

### PR DESCRIPTION
**Pull request by Nekotekina**
image
Something like this.

### CODE REVIEW

**Comment by Asinin3** [comment]
Huge thanks for this Neko. By the way is the "X" in "XFloat" supposed to be capitalized? I might have been typing it wrong all this time on our wiki lol.

**Comment by Nekotekina** [comment]
Capitalized or not, no real difference. It's a short for "extended float" which I made up.

File: rpcs3/rpcs3qt/settings_dialog.cpp [hunk 1]
```diff
@@ -206,18 +206,56 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> gui_settings, std
...
+		ui->xfloatAccuracy->setCurrentIndex(2);
+
+	connect(ui->xfloatAccuracy, QOverload<int>::of(&QComboBox::currentIndexChanged), ui->xfloatAccuracy, [this](int index)
+	{
+		if (index < 0) return;
+
+		if (ui->xfloatAccuracy->currentIndex() == 0)
+		{
+			m_emu_settings->SetSetting(emu_settings_type::AccurateXFloat, "true");
+			m_emu_settings->SetSetting(emu_settings_type::ApproximateXFloat, "false");
```
Comment (Megamouse) [inline rpcs3/rpcs3qt/settings_dialog.cpp:238]: why is there no setter for Relaxed?

Comment (Nekotekina) [reply rpcs3/rpcs3qt/settings_dialog.cpp:238]: There is no point in disabling it so it remains a hidden setting

**Review by Megamouse — APPROVED** [review]

**Pull request merged**